Workstream Policy.md

```diff
@@ -323,6 +323,15 @@ case-by-case basis by the [Steering Group].
 The repositories for all WHATWG publications and other deliverables
 must include the text of the applicable license in a LICENSE file.
 
+The contents of the license file should be
+> Copyright © YEAR WHATWG (Apple, Google, Mozilla, Microsoft).
+> This work is licensed under a Creative Commons Attribution 4.0 International License:
+
+Where YEAR is the current year, and followed by the full text of the
```

Comment:
lowercase "where" since it continues the sentence. And just "followed by" instead of "and followed by" sounds better?

Reply:
Hmm, when I make both these edits, it sounds like the full text of the license file should be filled in at YEAR. I'll see if I can make it two sentences.